src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
```

Comment:
there are no calculation inside, you can use without template literal, just `My name is {name}`

Reply:
I had written just `My name is {name}`, but I had a linter mistake
![image](https://github.com/mate-academy/react_person/assets/140401215/d0b30d48-3634-471e-9f9a-2586ab2c9e6b)

and in the page it looks without space
![image](https://github.com/mate-academy/react_person/assets/140401215/58dda5e0-7cb3-4bbe-b556-1a0d34955ade)
![image](https://github.com/mate-academy/react_person/assets/140401215/9419dfd4-c5bf-44a5-bc9b-663a5d2f52f6)